tests/e2e/login/purge.spec.js

```diff
@@ -248,7 +247,7 @@ describe('Purging on login', () => {
 
   const restartSentinel = () => utils.stopSentinel().then(() => utils.startSentinel());
 
-  it('Logging in as a restricted user with configured purge rules should not download purged docs', () => {
+  it('Logging in as a restricted user with configured purge rules should not download purged docs', async () => {
```

Comment:
As a general statement, I think adding await here is a bit confusing without reworking the entire promise chaining.  This sample code may be a little off as I'm writing it in the browser. Let me know if this needs a bit more clarity and we can jump on a call. 

```
let result = await getPurgeLog();
// purge ran but after initial replication, nothing to purge
chai.expect(result._rev).to.equal('0-1');
chai.expect(result.roles).to.equal(JSON.stringify(restrictedUser.roles.sort()));
chai.expect(result.history.length).to.equal(1);
chai.expect(result.count).to.equal(0);
chai.expect(result.history[0]).to.deep.equal({
          count: 0,
          roles: result.roles,
          date: result.date
        });
purgeDate = result.date;

await utils.resetBrowser();
await commonElements.calm();
await broweser.waitForAngular();
result = await getPurgeLog();
```

Reply:
fiexd